### DISCUSSION TITLE
Check for vulnerability in component newest version

### DISCUSSION THREAD

**Discussion by Trakeur** [DependencyTrack/dependency-track#2796] [Q&A] [answered]
Hi everyone,

Is there a way to quickly check for vulnerability in an unused component in projet ?

Because in the vulnerability search, it's not possible to search by component name, only by reference (i.e. unknown). 
I think it would be interesting to be able to quickly check whether a component version has a vulnerability before upgrading.

**Comment by syalioune** [comment, marked as answer]
With the current feature set of DT, the quickest would be to create a dummy project and add the component version to trigger vulnerability analysis.

**Reply by Trakeur** [reply]
Okay, that's what i did, thank you though.
